backend/src/main/java/com/cruru/applicant/service/EvaluationService.java

```diff
@@ -49,7 +56,11 @@ public void update(EvaluationUpdateRequest request, Evaluation evaluation) {
     }
 
     private boolean changeExists(EvaluationUpdateRequest request, Evaluation evaluation) {
-        return !(evaluation.getContent().equals(request.content()) && evaluation.getScore().equals(request.score()));
+        return !(
+                evaluation.getContent().equals(request.content())
+                        && evaluation.getScore().equals(request.score())
+                        && evaluation.getEvaluator().equals(request.evaluator())
```

Comment:
도메인 검증 로직으로 빼는건 어떤가요?

```suggestion
                evaluation.isContent() && evaluation.isScore() && evaluation.isEvaluator()
```

Reply:
저도 동의합니당